.github/workflows/wheels.yml

```diff
@@ -0,0 +1,96 @@
+name: Build Wheels
+
+on:
+  workflow_call:
+
+env:
+  CI_REGION: ${{ vars.S3_REGION }}
```

Comment:
Not sure if we need all of these, if I'm not missing something, looks like we're using just `${{ vars.S3_REGION }}`

Reply:
We need to exposure all of them to be able to run integration tests for S3 Standart and S3 Express.